src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/dec/BrotliStream.Decompress.cs

```diff
@@ -87,18 +107,56 @@ public override int Read(Span<byte> buffer)
             return totalWritten;
         }
 
+        /// <summary>Begins an asynchronous read operation. (Consider using the <see cref="System.IO.Stream.ReadAsync(System.Byte[],System.Int32,System.Int32)" /> method instead.)</summary>
+        /// <param name="buffer">The buffer from which data will be read.</param>
+        /// <param name="offset">The byte offset in <paramref name="array" /> at which to begin reading data from the stream.</param>
```

Comment:
This is a bug that came from the dotnet-api-docs text: the parameter `array` either used to exist in an old version of the method signature (and was changed before being shipped), or existed in .NET Framework.

```suggestion
        /// <param name="offset">The byte offset in <paramref name="buffer" /> at which to begin reading data from the stream.</param>
```

Reply:
> the parameter array either used to exist in an old version of the method signature (and was changed before being shipped), or existed in .NET Framework.

Yes, we renamed the parameter to be consistent with the base type.